susecloud

```diff
@@ -285,6 +285,16 @@ else
 fi
 
 #############################################################
+section_header "Ceph report"
+
+if [ -e /usr/bin/ceph ]; then
+    ceph_report_dir=$(mktemp -d)
```

Comment:
@vuntz , @aspiers : I added in https://github.com/toabctl/supportutils-plugin-susecloud/commit/911f2cc4d9500ed25ae5c37c15922ec40c877bd2 ```/var/log/supportconfig``` for now. And also added a timestamp to the file.